### PR TITLE
GitHub scripts

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

First example GitHub scripts.

**Why**:

We want to be able to script all kind of things and these scripts will be a good starting point.

**How**:

**Checklist**:

- [ ] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: scripts/github/repositories.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```
Comment (y0urself) [inline scripts/github/repositories.py:17]:
```suggestion

""" This script prints out all repositories existing in the space of the given organization """
```

File: scripts/github/repositories.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from argparse import ArgumentParser, Namespace
+
+from pontos.github.api import GitHubAsyncRESTApi
+from pontos.github.models.organization import Repository
+
+
+def add_script_arguments(parser: ArgumentParser) -> None:
+    parser.add_argument("organization")
```
Comment (y0urself) [inline scripts/github/repositories.py:25]:
```suggestion
    parser.add_argument("organization")
    
    parser.add_argument(
        "-t",
        "--type",
        help="what type of repositories should be printed? Default: %(default),
        default=RepositoryType.ALL
```

Maybe we can also add this arg, for e.g. only print public/private?

File: scripts/github/members.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```
Comment (y0urself) [inline scripts/github/members.py:17]:
```suggestion


""" This script prints out all members existing in the space of the given organization """
```
btw. what about outside colabs? are they printed, too?

File: scripts/github/repositories.py (hunk 2 above)
Comment (bjoernricks) [reply scripts/github/repositories.py:25]: Yes of course. This scripts are just a draft and starting point.

File: scripts/github/members.py (hunk 3 above)
Comment (bjoernricks) [reply scripts/github/members.py:17]: Haven't checked that yet. But i don't think so because there is an extra API for outside collabs.

**Review by y0urself — APPROVED** [review]

**Pull request merged**